src/main/java/seedu/tr4cker/logic/parser/ParserUtil.java

```diff
@@ -186,6 +186,26 @@ public static EventDate parseEventDate(String eventDate) throws ParseException {
         return new EventDate(trimmedEventDate, true);
     }
 
+    /**
+     * Parses a {@code String numDays} into an {@code int numDays}.
+     * Leading and trailing whitespace will be trimmed.
+     *
+     * @throws ParseException if the {@code numDays} is invalid.
+     */
+    public static int parseNumDays(String numDays) throws ParseException {
+        requireNonNull(numDays);
+        String trimmedNumDays = numDays.trim();
+        int numDaysInt;
+        try {
+            numDaysInt = Integer.parseInt(trimmedNumDays);
+        } catch (NumberFormatException nfe) {
+            throw new ParseException("Number of days must be a valid positive integer.");
+        }
+        if (numDaysInt < 0) {
+            throw new ParseException("Number of days must be a valid positive integer.");
```

Comment:
Maybe you can wrap this String as a variable in the Messages class since there're repetitive uses?